File: backend/team6/src/main/java/programmers/team6/domain/admin/support/VacationInfoLogs.java
package programmers.team6.domain.admin.support;

import java.util.Map;

import lombok.EqualsAndHashCode;
import programmers.team6.domain.vacation.entity.VacationInfoLog;

@EqualsAndHashCode
public class VacationInfoLogs {

	private final Map<Long, VacationInfoLog> lastedMap;

	public VacationInfoLogs(Map<Long, VacationInfoLog> lastedMap) {
		this.lastedMap = lastedMap;
	}

	public VacationInfoLog findVacationInfo(Long id) {
		return lastedMap.get(id);
	}
}

File: backend/team6/src/main/java/programmers/team6/domain/admin/support/VacationInfoLogs.kt
package programmers.team6.domain.admin.support

import lombok.EqualsAndHashCode
import programmers.team6.domain.vacation.entity.VacationInfoLog

@EqualsAndHashCode
class VacationInfoLogs(private val lastedMap: Map<Long, VacationInfoLog>) {
    fun findVacationInfo(id: Long): VacationInfoLog? = lastedMap[id]
}
